src/index.html

```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+
+      <div class="stars--row stars--0">
```

Comment:
```suggestion
      <div class="stars stars--0">
```
There is no need to use additional classes here